src/components/map/Map.tsx

```diff
@@ -331,18 +331,16 @@ export const Map = ({
                 />
               )}
 
-              {calloutTextEnabled && (
+              {calloutTextEnabled && title && (
```

Comment:
i cannot understand this. please read out loud in real language: "if callout text is enabled and there is a title, render a callout, that probably has a service name but always a title"

shouldn't it be more like: "if callout text is enabled and there is a title or a service name, render a callout with a service name a title or both"?

Reply:
I only added title control because we have title value in other parts of the app but we have serviceName value only in Sue module. Therefore, I thought of adding only title control to make it work smoothly in other parts of the app. But what you're saying makes more sense. I will update it this way:
```
{calloutTextEnabled && (!!title || !!serviceName)...
```

Yes, you are right, I need to convert it to a boolean expression.